netbox/data_source_netbox_tenants_test.go

```diff
@@ -65,3 +65,39 @@ data "netbox_tenants" "test" {
 		},
 	})
 }
+
+func TestAccNetboxTenantGroupDataSource_filter(t *testing.T) {
```

Comment:
The canonical name for these test functions is
`TestAcc<Name><"DataSource" for data sources">_<test description>`

In this example, it should be
`TestAccNetboxTenantsDataSource_whatever`, e.g. `TestAccNetboxTenantsDataSource_tenantgroups`

Reply:
Test is renamed and pushed